prefect_email/credentials.py

```diff
@@ -133,7 +135,9 @@ def example_get_server_flow():
         if smtp_type == SMTPType.INSECURE:
             server = SMTP(smtp_server, smtp_port)
         else:
-            context = ssl.create_default_context()
+            context = self.context
+            if context is None:
+                context = ssl.create_default_context()
```

Comment:
```suggestion
            context = self.context or ssl.create_default_context()
```

Reply:
Can we add a test for this; e.g. passing context as an argument, [here](https://github.com/PrefectHQ/prefect-email/blob/main/tests/test_message.py#L46-L55)? And if there's a way to assert that the context was set?

Reply:
Sure, I'll add it

Reply:
Adding "ssl_context: Optional[ssl.SSLContext]" as a parameter to email_send_message() is going to result in all the pydantic related issues again. It really needs to be Optional[object] and then proper documentation in the function docstring.

Or I cold just unwind all this and use my own EmailServerCredentials....  But someone might want this at some point - so tell me what to do here

Reply:
See this comment:
https://github.com/PrefectHQ/prefect-email/pull/38#discussion_r989395382